### DISCUSSION TITLE
Bad request reading js chunk

### DISCUSSION THREAD

**Discussion by MartinKuhne** [ChilliCream/hotchocolate#5030] [General]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

Banana Cake Pop fails to load (blank screen). There is a 400 status code returned for one of the js chunks

On the server, I have these two log entries **for the same correlation id**

```
OnError: {"Message":"Either the parameter query or the parameter id has to be set.","Code":"HC0013","Path":null,"Locations":null,"Extensions":[{"Key":"code","Value":"HC0013","$type":"KeyValuePair`2"}],"Exception":null,"SyntaxNode":null,"$type":"Error"}
HTTP GET /graphql/static/js/18.6378174f.chunk.js responded 400 in 2112.3697 ms
```

The first one is generated from instrumentation I have in an IErrorFilter OnError callback. The other one is generated by the asp.net core runtime.

I am not sure where to start looking. We should not be in the query execution path for static files.

### Steps to reproduce

1. Go to <url>/graphql

### Product

Banana Cake Pop

### Version

12.7.0

**Comment by MartinKuhne** [comment]
I can retrieve the offending chunk from the server individually
Other chunks load successfully

**Comment by michaelstaib** [comment]
Is the issue still existing with 12.8.2?

**Reply by MartinKuhne** [reply]
updated, didn't fix the issue

**Comment by michaelstaib** [comment]
It could be a browser cache issue.

**Reply by MartinKuhne** [reply]
I tried different browsers... also several people on my team have the same issue
